py/objint_longlong.c

```diff
@@ -40,14 +40,6 @@
 
 #if MICROPY_LONGINT_IMPL == MICROPY_LONGINT_IMPL_LONGLONG
 
-// Python3 no longer has "l" suffix for long ints. We allow to use it
```

Comment:
Ah, so I finally got the "unused anywhere" part, sorry. Still, this shows why unrelated changes should be included in one commit - this spurs confusion.

> Anyway: proper fix is then to also introduce e.g. MICROPY_DEBUG_LONGINT ?

Well, so I looked into this just to not let it linger for months - we do have technical debt, so it can only be appreciated when non-maintainers try to decrease it. But: "proper" way to fix needs to be considered for months, because it's wide issue, and not a priority at all. I can tell you an anecdote that I was unpleasantly surprised to see in Contiki the situation when you need to go and patch a particular file to enable debugging there. Fast forward few years, and we're using the same "anti-pattern" in uPy. That's probably because this "anti-pattern" is awfully practical, and any alternative while solves something, brings in new issues, so even the final balance isn't clear, and that's what needs consideration for months.

Anyway, to be done with it, please fix typo in commit message, and let's have this merged.